gctscript/modules/gct/errors.go

```diff
@@ -29,3 +30,10 @@ func errorResponsef(format string, a ...interface{}) (objects.Object, error) {
 		Value: &objects.String{Value: fmt.Sprintf(format, a...)},
 	}, nil
 }
+
+func constructRuntimeError(argPosition int, funcName, expectedType string, whatYouGot interface{}) error {
```

Comment:
`whatYouGot` 😄

Reply:
also please add dedicated test. Probably only need to just verify its return of a `common.ErrTypeAssertFailure`